dev/com.ibm.jbatch.container/src/com/ibm/jbatch/container/ws/BatchGroupSecurityHelper.java

```diff
@@ -0,0 +1,18 @@
+package com.ibm.jbatch.container.ws;
+
+import java.util.List;
+
+import javax.security.auth.Subject;
+
+public interface BatchGroupSecurityHelper {
+
+    /**
+     * @return Returns the list of group names
+     */
+    public List<String> getGroupsForSubject(Subject subject);
```

Comment:
Maybe a bit more detail  "group names of groups the subject userid is a member of".  Describe boundary case (null, vs empty list).   If there's no ordering here, Set probably better.